client/src/hooks/reducers/authFormReducer.js

```diff
@@ -19,6 +19,7 @@ export const authFormReducer = (state, action) => {
   switch (type) {
     case AUTH_FORM_LOGIN:
     case AUTH_FORM_SIGNUP:
+      return { ...state, email: payload.email };
```

Comment:
What's up with this?

Reply:
I needed to access email from state to display it on the page. In the login component we were saving email to state. Wanted explicitly return it here.